packages/account/PasskeyAccount/passkeyOwnerWebAuthn/background/webAuthn.ts

```diff
@@ -0,0 +1,214 @@
+import { runtime, tabs, windows, type Runtime } from "webextension-polyfill"
+
+import { PortName } from "~packages/account/PasskeyAccount/passkeyOwnerWebAuthn/tabs/port"
+import json from "~packages/util/json"
+import {
+  isWebAuthnError,
+  type WebAuthnAuthentication,
+  type WebAuthnError,
+  type WebAuthnRegistration
+} from "~packages/webAuthn/typing"
+
+/**
+ * Asynchronously creates a new window and sets up listeners for port communication.
+ * @param {string} createWindowUrl - The URL for creating the new window.
+ * @returns {Promise<chrome.windows.Window>} A Promise that resolves to the created window.
+ */
+export const webAuthnWindowAsync = async (
+  createWindowUrl: string
```

Comment:
It would be better to fix the tab url in this function, and make this function explicitly accept 
 required arguments, something like:

```typescript
const webAuthnWindowAsync = async (credentialId, challenge) 
```

Or other developers would not know what are the required parameters to construct the right url.